CONTRIBUTING.md

```diff
@@ -226,6 +226,15 @@ e.g. run tests against v 1.0
 
 If the "--api-version" argument is not specified, all runs act against default api version for each tested command.
 
+
+### IoT Digital Twins
+
+IoT Digital Twins test for creation of larger ontologies require ontology submodules to be cloned.
```

Comment:
is this a manual step? What if I don't want to or forget - do the tests not run or do I error?

Reply:
Yes, the user will need to run a command to fetch the submodules. The part of the tests needing the submodules are skipped in case the submodules are not checked out.

Reply:
btw where is the pipeline task for this? Does the run parallel tests actually do it? (also is the checkout only done for dt tests?)

Reply:
Yes, the pipeline task for running tests checks out the submodules as well - the change is part of this PR. The checkout happens by default for all jobs on the pipeline - my change ensures that apart from the repository the submodules are also checked out for **all test jobs**. The submodule checkout takes additional few seconds - but allows us to be future proof - means going forward any of our tests can utilize submodules without making updates to the pipeline.